test/operators.js

```diff
@@ -780,6 +780,34 @@ prepareAndRunTest('support empty deferred operations', dir, (t, db, raf) => {
   })
 })
 
+prepareAndRunTest('empty deferred AND equal', dir, (t, db, raf) => {
```

Comment:
This test actually passes with the previous implementation, it was just good to add anyway.